bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewerCommand.java

```diff
@@ -0,0 +1,136 @@
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.skara.bots.pr;
+
+import org.openjdk.skara.census.Contributor;
+import org.openjdk.skara.forge.PullRequest;
+import org.openjdk.skara.issuetracker.Comment;
+
+import java.io.PrintWriter;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class ReviewerCommand implements CommandHandler {
+    private static final Pattern commandPattern = Pattern.compile("^(add|remove)\\s+(.+)$");
+
+    private void showHelp(PullRequest pr, PrintWriter reply) {
+        reply.println("Syntax: `/reviewer (add|remove) [@user | openjdk-user]`. For example:");
+        reply.println();
+        reply.println(" * `/reviewer add @openjdk-bot`");
+        reply.println(" * `/reviewer add duke`");
+    }
+
+    private Optional<Contributor> parseUser(String user, PullRequest pr, CensusInstance censusInstance) {
+        user = user.strip();
+        if (user.isEmpty()) {
+            return Optional.empty();
+        }
+        Contributor contributor;
+        if (user.charAt(0) == '@') {
+            var platformUser = pr.repository().forge().user(user.substring(1));
+            if (platformUser.isEmpty()) {
+                return Optional.empty();
+            }
+            contributor = censusInstance.namespace().get(platformUser.get().id());
+            if (contributor == null) {
+                return Optional.empty();
+            }
+        } else {
+            contributor = censusInstance.census().contributor(user);
+            if (contributor == null) {
+                return Optional.empty();
+            }
+        }
+        return Optional.of(contributor);
+    }
+
+    @Override
+    public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, String args, Comment comment, List<Comment> allComments, PrintWriter reply) {
+        if (!comment.author().equals(pr.author())) {
+            reply.println("Only the author (@" + pr.author().userName() + ") is allowed to issue the `reviewer` command.");
+            return;
+        }
+        if (bot.ignoreStaleReviews()) {
+            reply.println("This project requires authenticated reviews - please ask your reviewer to flag this PR as reviewed.");
```

Comment:
Yep, but all command replies are automatically prepended with an `@comment-author` - and since the check above ensures that the comment author is the pr-author the message should only happen as a reply to the author of the pr.